R/render.R

```diff
@@ -54,6 +54,10 @@ render <- function(input,
                        encoding = encoding)
       outputs <- c(outputs, output)
     }
+    if (length(output_file) > 1) {
```

Comment:
Should we verify that the length of output_file is appropriate for the number of output formats? (or does recycling take care of doing the right thing even when the number is different?)